packages/sdk/README.md

```diff
@@ -1,11 +1,15 @@
 # GroupFi Chatbox SDK
 
-GroupFi Chatbox SDK enables developers to easily integrate GroupFi's chatbox with their dApps on EVM chains through popular wallet extensions, including `MetaMask`, `OKX Wallet`, and `Trust Wallet`.
+GroupFi Chatbox SDK enables developers to easily integrate GroupFi's chatbox with their dApps on EVM chains, Solana through popular wallet extensions, including `MetaMask`, `OKX Wallet`, `Phantom Wallet`, `Trust Wallet` and so on.
```

Comment:
with their EVM and Solana dApps either directly through wallet extensions, including..., or popular wallet wrappers such as WalletConnect, Rainbowkit and Rabbykit.

Reply:
这个改动我有点把握不住:sweat_smile:，我原本想表达的是：chatbox 支持什么 chains, 支持哪些钱包。
这里的 EVM dapps, Solana Dapps 我不太懂是不是包含我想表达的。
Rainbowkit、Rabbykit 这些涉及集成的一些包需要写出来吗？WalletConnect 我们不支持

Reply:
I think it would be obvious that GroupFi supports EVM and Solana chains by saying EVM and Solana dApps. I want to mention Rainbowkit, Rabbykit in the front since many projects are using them instead of individual plugins